src/index.css

```diff
@@ -54,8 +54,12 @@ body {
   max-width: 42rem;
   background-color: var(--color-background-500);
   border-radius: 0.9rem;
-  /* overflow: scroll; */
+  overflow: scroll;
+  overflow-x: hidden;
   position: relative;
+
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
```

Comment:
```suggestion
  overflow: hidden
```

Reply:
by setting `overflow:hidden;` the scrolling is disabled that's why on line 57 i had used `overflow: scroll;` and on line 58 to prevent showing horizontal scroll
and line 61 is to hide the scroll bar as it was not looking good